docs/examples/testing/test_health_check_sync.py

```diff
@@ -0,0 +1,31 @@
+import pytest
```

Comment:
Okay so you've got this tested now, but it's still just duplicated from the docs. What remains is to replace the `.. code-block::` in the documentation file this is taken from with a `.. literalinclude:: /examples/testing/test_health_check_sync.py`. This applies to `/examples/testing/test_health_check_async.py` as well.